packages/renderer/src/AppNavigation.svelte

```diff
@@ -89,7 +89,8 @@ export let meta;
 
 <svelte:window />
 <nav
-  class="pf-c-nav z-0 group w-[54px] min-w-[54px] flex flex-col justify-between overflow-hidden hover:overflow-y-auto"
+  class="pf-c-nav z-0 group w-[54px] min-w-[54px] flex flex-col justify-between overflow-hidden hover:overflow-y-auto top-0"
+  style="position: sticky"
   aria-label="Global">
```

Comment:
'sticky' does not work ?

Reply:
`pf-c-nav` class override it with `relative`